api/app/signals/apps/dataset/management/commands/load_gebieden.py

```diff
@@ -0,0 +1,93 @@
+import requests
+from django.contrib.gis.geos import LinearRing, MultiPolygon, Polygon
+from django.core.management import BaseCommand
+
+from signals.apps.signals.models import Area, AreaType
+
+
+class GebiedenAPIGeometryLoader:
+    """Convert not-quite-GeoJSON output from Gebieden API to GEOS geometries."""
+
+    @classmethod
+    def load_polygon(cls, rd_coordinates):
+        linear_rings = []
+        for entry in rd_coordinates:
+            lr = LinearRing(entry, srid=28992)
+            linear_rings.append(lr)
+        return Polygon(*linear_rings, srid=28992)
+
+    @classmethod
+    def load_multipolygon(cls, rd_coordinates):
+        polygons = []
+        for entry in rd_coordinates:
+            polygons.append(cls.load_polygon(entry))
+        return MultiPolygon(polygons, srid=28992)
+
+
+class APIGebiedenLoader:
+    """
+    Load entries from Datapunt "Gebieden API", save as SIA Area instances.
+    """
+    GEBIEDEN_URL = 'https://api.data.amsterdam.nl/gebieden/{}/'
+    CODE_FIELDS = {
```

Comment:
Same as above, maybe this should be configurable?

Reply:
See answer above, this is not generic code.